logger.go

```diff
@@ -25,6 +26,12 @@ func (l *Logger) AddTransport(t ...*Transport) *Logger {
 	return l
 }
 
+func (l *Logger) SetDefaultMeta(meta map[string]string) *Logger {
```

Comment:
A logger should be non modifiable, thus don't use this function and instead rely on the New(...) function.